exchange/bidder_test.go

```diff
@@ -1254,15 +1256,28 @@ func TestTimeoutNotificationOff(t *testing.T) {
 	if tb, ok := bidder.Bidder.(adapters.TimeoutBidder); !ok {
 		t.Error("Failed to cast bidder to a TimeoutBidder")
 	} else {
-		bidder.doTimeoutNotification(tb, &adapters.RequestData{})
+		bidder.doTimeoutNotification(tb, &adapters.RequestData{}, glog.Warningf)
 	}
 }
 
 func TestTimeoutNotificationOn(t *testing.T) {
+	ctx, cancelFunc := context.WithDeadline(context.Background(), time.Now().Add(-7*time.Hour))
+	cancelFunc()
 	respBody := "{\"bid\":false}"
 	respStatus := 200
-	server := httptest.NewServer(mockHandler(respStatus, "getBody", respBody))
+	server := httptest.NewServer(mockSlowHandler(respStatus, "getBody", respBody))
```

Comment:
The slow handler waits 500ms. I think that's too long for a unit test. Ideally we'd inject a mock context to the timeout caller, but since we can't I think using smaller timeouts is a good compromise. Let's use 201ms or 205ms since the method is hardcoded for 200ms.